scrapy/extensions/feedexport.py

```diff
@@ -591,7 +591,7 @@ def _settings_are_valid(self):
 
     def _storage_supported(self, uri, feed_options):
         scheme = urlparse(uri).scheme
-        if scheme in self.storages:
+        if scheme in self.storages or PureWindowsPath(uri).drive:
```

Comment:
```
>>> from urllib.parse import urlparse
>>> urlparse("C:\Scrapy\output.json").scheme
'c'
```
Had to add this condition because `scheme` for windows absolute paths is misleading.